modules/distribution/src/repository/resources/dbscripts/apimgt/mssql.sql

```diff
@@ -846,10 +855,10 @@ CREATE TABLE AM_API_SCOPES (
 IF NOT  EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[AM_API_DEFAULT_VERSION]') AND TYPE IN (N'U'))
 
 CREATE TABLE AM_API_DEFAULT_VERSION (
-            DEFAULT_VERSION_ID INTEGER NOT NULL IDENTITY, 
+            DEFAULT_VERSION_ID INTEGER NOT NULL IDENTITY,
             API_NAME VARCHAR(256) NOT NULL ,
-            API_PROVIDER VARCHAR(256) NOT NULL , 
-            DEFAULT_API_VERSION VARCHAR(30) , 
+            API_PROVIDER VARCHAR(256) NOT NULL ,
+            DEFAULT_API_VERSION VARCHAR(30) ,
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04